### PR TITLE
Add missing history codes

### PR DESCRIPTION
**Pull request by Ben-Edwards-cgi**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/CompleteServiceControllerITest.java [hunk 1]
```diff
@@ -73,12 +73,37 @@ public void setUp() throws Exception {
         httpHeaders.setAccept(Collections.singletonList(MediaType.APPLICATION_JSON));
     }
 
+
+    private void validateJurorWasCompleted(LocalDate completionTime, String jurorNumber, String poolNumber,
+                                           boolean isDismissal) {
+        JurorPool jurorPool = jurorPoolRepository.findByJurorJurorNumberAndPoolPoolNumber(jurorNumber, poolNumber);
+        assertEquals(true, jurorPool.getIsActive(),
+            "Juror pool should be inactive");
```
Comment (philhead1) [inline src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/CompleteServiceControllerITest.java:81]: Message states pool should be inactive but assertion is to see if the juror pool is active. Either the message needs updating or the true value should be flipped to false.

File: src/main/java/uk/gov/hmcts/juror/api/moj/domain/FormCode.java [hunk 2]
```diff
@@ -13,8 +13,8 @@ public enum FormCode {
     BI_EXCUSAL("5225C", PrintDataService::printExcusalLetter, IJurorStatus.EXCUSED),
     ENG_EXCUSALDENIED("5226", PrintDataService::printExcusalDeniedLetter, IJurorStatus.RESPONDED),
     BI_EXCUSALDENIED("5226C", PrintDataService::printExcusalDeniedLetter, IJurorStatus.RESPONDED),
-    ENG_SUMMONS("5221", PrintDataService::printSummonsLetter, IJurorStatus.SUMMONED),
-    BI_SUMMONS("5221C", PrintDataService::printSummonsLetter, IJurorStatus.SUMMONED),
+    ENG_SUMMONS("5221", PrintDataService::reprintSummonsLetter, IJurorStatus.SUMMONED),
+    BI_SUMMONS("5221C", PrintDataService::reprintSummonsLetter, IJurorStatus.SUMMONED),
```
Comment (philhead1) [inline src/main/java/uk/gov/hmcts/juror/api/moj/domain/FormCode.java:17]: I don't believe these should be reprint as they are just a summons letter and have the ability to be reprinted.

**Review by philhead1 — COMMENTED** [review]
just a couple of small comments, but aside from that all good 😸

File: src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/CompleteServiceControllerITest.java (hunk 1 above)
Comment (Ben-Edwards-cgi) [reply src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/CompleteServiceControllerITest.java:81]: Updated

File: src/main/java/uk/gov/hmcts/juror/api/moj/domain/FormCode.java (hunk 2 above)
Comment (Ben-Edwards-cgi) [reply src/main/java/uk/gov/hmcts/juror/api/moj/domain/FormCode.java:17]: Reprinted letters have a different history code added then initial printing :)

**Review by philhead1 — APPROVED** [review]

**Pull request merged**